server/bin/pbench-verify-backup-tarballs

```diff
@@ -262,11 +262,16 @@ def compare_entry_lists(list_one_obj, list_two_obj, report, logger):
 
 def sanity_check(s3_obj, logger):
     # make sure the S3 bucket exists
+    if s3_obj.bucket_name is None:
+        logger.warning("Bucket not defined in config file - S3 backup is disabled.")
```

Comment:
Doesn't really sound like a warning; it just means we're not doing S3 backup, right? `info` would probably be more appropriate.

Reply:
I kept it the same as what `pbench-backup-tarballs` is doing.  If you feel strongly, I can change both, but I'd rather not have them different.

The reason why it is a warning is because it is dealing with backups.  If we aren't getting backups because of a bucket missing, it seems like that is a concern.